src/main/java/mate/controller/driver/LoginController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+@WebServlet(urlPatterns = "/login")
+public class LoginController extends HttpServlet {
+    public static final String SESSION_ATTRIBUTE = "driver_id";
+    private static final Injector injector = Injector.getInstance("mate");
+    private final AuthenticationService authenticationService = (AuthenticationService) injector
+            .getInstance(AuthenticationService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/drivers/login.jsp").forward(req,resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        try {
+            Driver driver = authenticationService.login(login, password);
+            HttpSession session = req.getSession();
+            session.setAttribute(SESSION_ATTRIBUTE, driver.getId());
```

Comment:
в нас може бути багато атрибутів різних
```suggestion
            session.setAttribute(SESSION_ATTRIBUTE_ID, driver.getId());
```